overrides.json

```diff
@@ -94,8 +101,7 @@
   "@material-ui/utils": {
     "4": {
       "exports": {
-        ".": "./index.js",
-        "./index.js": {
+        ".": {
           "require": "./index.js",
           "import": "./esm/index.js"
         },
```

Comment:
```suggestion
        },
        "./index.js": "./index.js",
```

Reply:
ok, so if index.js is explicitly required then we explicitly provide that rather than the esm override?

Reply:
Yes because that's the expectation since without exports that's what they'd get before.